### PR TITLE
Add custom value for coin drops when idle

### PR DESCRIPTION
**Pull request by michaelowens**
Allows the user to configure a different coin drop value for idlers/lurkers.

### CODE REVIEW

File: models/Channel.iced [hunk 1]
```diff
@@ -341,11 +341,12 @@ class exports.Channel extends Mikuia.Model
 			callback err, false
 
 	isSupporter: (callback) ->
-		await @getSupporterStatus defer err, data
-		if data > (new Date()).getTime() / 1000
-			callback err, true
-		else
-			callback err, false
+		callback false, true
```
Comment (Maxorq) [inline models/Channel.iced]: uhh

**Comment by michaelowens** [comment]
I need to revert all the crap I just committed which wasn't intended to be committed. My bad for doing `git commit -am`

**Comment by Maxorq** [comment]
Yeeaaah.

**Comment by michaelowens** [comment]
Here we go, that looks more like the changes that it should have.

**Comment by jariz** [comment]
Worst open source contributor 2015

**Comment by michaelowens** [comment]
As you can see I'm great at doing these pull requests.

**Pull request closed without merging**